### DISCUSSION TITLE
Applying box count method to array of stock data

### DISCUSSION THREAD

**Discussion by laclimber** [PMEAL/porespy#793] [Q&A]
I want to apply the box count method to the last 20 years of stock price data. I have an array with shape 4946 X 2 in numpy form. But when I apply the box count function to it, I don't get any reasonable results. Just this:
![Screenshot_20230221_120755](https://user-images.githubusercontent.com/126019483/220447670-b10572d9-ba16-4451-82f0-619c9f04e060.png)

**Comment by jgostick** [comment]
Is the image type boolean?  And what does it look like?

**Reply by laclimber** [reply]
the array is type numpy.ndarray . and looks like this:
array([[0.00000000e+00, 6.77971734e+00],
[1.00000000e+00, 6.81237810e+00],
[2.00000000e+00, 6.81189395e+00],
...,
[4.94300000e+03, 8.47492432e+00],
[4.94400000e+03, 8.47193010e+00],
[4.94500000e+03, 8.46930042e+00]])

I put the price in log form

**Reply by jgostick** [reply]
Im sorry I didn't realize you were doing time data.  The function was made for boolean images, and is definitely not suitable for your case.

**Reply by laclimber** [reply]
i was able to convert into a boolean image (from the picture of the chart) and got the box count method to work. But how do i find the exact fractal dimension?
![image](https://user-images.githubusercontent.com/126019483/220761186-9eb6e82c-82c6-4eb1-a648-09828a48ecac.png)

**Reply by laclimber** [reply]
![image](https://user-images.githubusercontent.com/126019483/220761270-ac7cdb0a-0de1-4233-b1fd-123d4b32093c.png)

**Reply by jgostick** [reply]
It is the slope of the line on the left, which is what is plotted on the right.  Seems to be about 1, so not very fractal.
